Extensions/dnSpy.StringSearcher/StringReferencesToolWindowContent.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Windows;
+using dnSpy.Contracts.ToolWindows;
+using dnSpy.Contracts.ToolWindows.App;
+
+namespace dnSpy.StringSearcher {
+	[Export(typeof(IToolWindowContentProvider))]
+	sealed class StringReferencesToolWindowContentProvider : IToolWindowContentProvider {
+		private readonly Lazy<IStringReferencesService> service;
+
+		public StringReferencesToolWindowContent DocumentTreeViewWindowContent
+			=> toolWindowContent ??= new StringReferencesToolWindowContent(service);
+		StringReferencesToolWindowContent? toolWindowContent;
+
+		[ImportingConstructor]
+		StringReferencesToolWindowContentProvider(Lazy<IStringReferencesService> stringSearcher) {
+			this.service = stringSearcher;
+		}
+
+		public IEnumerable<ToolWindowContentInfo> ContentInfos {
+			get {
+				yield return new ToolWindowContentInfo(
+					StringReferencesToolWindowContent.THE_GUID,
+					StringReferencesToolWindowContent.DEFAULT_LOCATION,
+					AppToolWindowConstants.DEFAULT_CONTENT_ORDER_BOTTOM_ANALYZER,
+					false
```

Comment:
The `isDefault` boolean parameter is optional and defaults to `false`. Explicitly setting it to `false` is unecessary.